src/components/Drum.tsx

```
@@ -1,32 +1,50 @@
+
 export const DrumComponents = () => {
+  const pressedKeys = useRef<Set<string>>(new Set())
   const play = (sound: string) => {
     const audio = new Audio(`${SOUND_URL}/${sound}.mp3`)
     audio.play().catch((error) => {
       console.error(`사운드 재생 오류 ${sound}:`, error)
     })
   }
 
+  useEffect(() => {
+    const audios: HTMLAudioElement[] = SOUND_LIST.map((sound) => {
```

ESLint

[error] 'audios' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)